schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-associated.xsl

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:mdb="http://standards.iso.org/iso/19115/-3/mdb/2.0"
+                xmlns:mri="http://standards.iso.org/iso/19115/-3/mri/1.0"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+                xmlns:pav="http://purl.org/pav/"
+                xmlns:dct="http://purl.org/dc/terms/"
+                exclude-result-prefixes="#all">
+
+  <!-- TODO: either we only use the associated records in the current record
+             either we get all associated records using the API?
+             we may also have limitation depending on user privileges.-->
+
+  <xsl:variable name="isoAssociatedTypesToDcatCommonNames"
+                as="node()*">
+    <entry associationType="crossReference" initiativeType="collection">dct:isPartOf</entry>
+    <entry associationType="partOfSeamlessDatabase">dct:isPartOf</entry>
+    <entry associationType="series">dct:isPartOf</entry>
+    <!-- TO DISCUSS: Should we use dcat:inSeries instead of isPartOf -->
```

Comment:
Why is not used `dcat:inSeries`?

Reply:
To be discussed :). There is no specific guideline on this ?